hazelcast-enterprise/log4j2.properties

```diff
@@ -0,0 +1,15 @@
+# Default Logging Configuration
+
+appender.console.type=Console
+appender.console.name=STDOUT
+appender.console.layout.type=PatternLayout
+appender.console.layout.pattern=%d [%highlight{${LOG_LEVEL_PATTERN:-%5p}}{FATAL=red, ERROR=red, WARN=yellow, INFO=green, DEBUG=magenta}] [%style{%t{1.}}{cyan}] [%style{%c{1.}}{blue}]: %m%n
+
+rootLogger.level=info
+rootLogger.appenderRef.stdout.ref=STDOUT
+
+# logger.internal.name=com.hazelcast.internal
```

Comment:
I wonder if we need this commented-out lines. What are they for?

Reply:
They were examples of custom configurations like we currently have:
https://github.com/hazelcast/hazelcast-docker/blob/4be9a22304186153a8f4fc55b1ef661875c47bd4/hazelcast-oss/logging.properties#L51-L58
However, these are log4j2 related properties, and including only a few examples does not make much sense as there are many others. Hence I removed completely.